### PR TITLE
update pull quote styles

### LINKED ISSUE #531: Pull quote font sizes is incorrect
# 1. Bug template

## Process

1. Navigate to patterns > pull quote

## Expected result

The font sizes should be 23px for the text, 45px for the quotation marks, and 20px italic for the cite.

The spacing between the quote and the cite might need adjusting instead of 12px to 18px or 24px.

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* The font sizes should be 23px for the text, 45px for the quotation marks, and 20px italic for the cite.
* The spacing between the quote and the cite is 16px
* updated the copy doc

## QA

1. go to /patterns/pull-quote/ and see that these font-sizes and margin are correct

## Details

Fixes #531 and and [vanillaframework.io bug 32](https://github.com/ubuntudesign/vanillaframework.io/issues/32)

## Screenshots

image

### CODE REVIEW

**Comment by barrymcgee** [comment]
@pmahnke Can you fix build error please?

**Comment by pmahnke** [comment]
@barrymcgee I fixed the whitespace error

File: scss/_patterns_pull-quotes.scss [hunk 1]
```diff
@@ -3,12 +3,28 @@
...
     max-width: 75%;
-    font-size: 1.5rem;
+
+    > p {
+      font-size: 1.4375rem;
+
+      &:first-of-type::before,
+      &:last-of-type::after {
+        font-size: 2.8125rem;
+        margin: 4em -.6rem;
```
Comment (barrymcgee) [inline scss/_patterns_pull-quotes.scss]: You have an `em` here, please change to `rem`

**Comment by barrymcgee** [comment]
@pmahnke Did you spot the `em` to `rem` note above?

**Pull request closed without merging**